app/controllers/application_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ApplicationController < ActionController::Base
+  include Pundit
   before_action :authenticate_user!
 
```

Comment:
We want to call `after_action :verify_authorized, unless: :devise_controller?` and `after_action :verify_policy_scoped, only: :index, unless: :devise_controller?`. This is to make sure that we have set up policies for every action. It may break things, but it is telling us we missed policies.